core-kotlin-modules/core-kotlin-strings-3/src/test/kotlin/com/baeldung/palindromicstring/PalindromicStringTest.kt

```diff
@@ -0,0 +1,60 @@
+package com.baeldung.palindromicstring
+import org.junit.jupiter.api.Test
+
+class PalindromicStringTest {
+
+    @Test fun `palindrome check using loop function`(){
+        assert(isPalindromeUsingLoop("racecar") == true)
```

Comment:
For all trivial assertions, prefer either the JUnit5's assertTrue, assertFalse, etc, or use the AssertJ library's fluent assertions.